data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,734 @@
+	choice
+		`	(Check your pocket for anything unexpected.)`
+	`	As expected, you find something unexpected: a small device about the size of a fingernail. You gingerly remove it from your belongings and place it on the table. The officer across from you wastes no time in drawing his pistol and sharply smashing its butt against the device, crushing it to pieces.`
+	`	"Care to explain what that was doing on you?"`
+	label explanation
+	choice
+		`	"I didn't bring that here on purpose, but I did brush against a very friendly janitor on the way here."`
+	`	Ahhs of understanding and enlightenment echo from the soldiers standing about the hangar. Most of them visibly relax, appeased by this explanation.`
+	`	"Yeah. That's Frank. Republic Intelligence. We'd shoot him, but they would just send another, and he's incompetent enough that it's better this way. Really, though? You need to be more careful."`
+	`	You hear a barely-audible "jackass" muttered by one of the soldiers. The officer doesn't appear to have any desire to reprimand them.`
+	branch continued
+		not "clobbered"
+	`	Given a nod from their commander, one of the soldiers hunkers down behind your chair to cut your bonds loose.`
+	label continued
+	`	A soldier emerges from the ramp to the ancient hauler, a neon yellow object cradled close to her burgundy coat. It's held out for you to take.`
+	`	"This is what you came for," the officer explains. "If it's to your satisfaction you can leave with it, and we'll keep that credit chip."`
+	choice
+		`	(Inspect the object.)`
+	`	It's a steel canister about the size of a thermos. The brilliant yellow appearance from a distance proves to be thanks to a plethora of warning labels stuck all over the canister. An unsettling quantity of stylized skulls and exclamation points are visible, leading to an almost polka-dotted appearance if you squint. Present in the middle of the warning labels is a simple placard welded to the canister. It reads, plainly: "Drought-Resistant Seeds". No other documentation as to what's actually in the can seems to be forthcoming.`
+	choice
+		`	"Looks good to me, thank you."`
+			goto complete
+		`	"What in the void is this?"`
+	`	The dark-haired officer simply smiles gently. "It is what it is. Now take it and run along to your employer. We're done here."`
+	branch complete
+		not "clobbered"
+	`	The soldiers give you a moment to dress yourself and gather your things back up, though the surly looks you're fixed with make it clear they're keen to see you go.`
+	label complete
+	action
+		payment -1000000
+	`	The transaction is complete. You've spent a million credits out of your own pocket for something that needs to be delivered to a mysterious mercenary you met in a stinking bar. This is definitely the path of wisdom that you walk. Thankfully, no further obstacles ambush you as you return to your ship.`
```

Comment:
`This is definitely the path of wisdom that you walk.`
Banger of a line. Made me chuckle.